.github/workflows/release_linux.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2010_x86_64
       with:
         python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39'
-        build-requirements: 'numpy'
+        build-requirements: 'numpy protobuf==3.11.3'
```

Comment:
You need to set numpy version as well.

If a package was built with numpy version A, then at runtime it requires numpy version >=A.

Reply:
Do you have any recommended version? I don't see any numpy requirement in ONNX so I don't know which version is appropriate...

Reply:
I don't have. But you may ask the people in sigs.